administration/src/util/normalizeString.ts

```diff
@@ -1,6 +1,11 @@
-import normalizeStrings from 'normalize-strings'
+// eslint-disable-next-line no-control-regex
+const nonAsciiRegex = /[^\x00-\x7F]/g
 
-const normalizeString = (str: string): string => normalizeStrings(str).toLowerCase().trim()
-export const normalizeName = (name: string): string => normalizeString(name).replaceAll(/[^a-zA-Z0-9]/g, '')
+const normalizeToAscii = (str: string): string => str.normalize('NFKD').replace(nonAsciiRegex, '')
```

Comment:
:question: Why do we want the decomposed form here?

Reply:
To convert, for example, "ä" to "a+modifier". (this just resembles the previous implementation of `normalize-strings`.

Reply:
Ahhh makes sense, I did not get this from the docs. Thanks for explaining.